docs/overview/contributing/setting_up.rst

```diff
@@ -694,6 +694,69 @@ Note: Currently you do not need to comment out the :code:`conftest.py` file in t
     </iframe>
 
 
+Setting up with WSL
+-------------------
+
+Here is another way for you to set up the whole Ivy environment on Windows using WSL, virtual environment and VSCode. 
```

Comment:
Also please remove this first line, we don't need to specify this type of stuff here. :slightly_smiling_face: 
`
Here is another way for you to set up the whole Ivy environment on Windows using WSL, virtual environment and VSCode. `